spk/tt-rss/Makefile

```diff
@@ -1,6 +1,6 @@
 SPK_NAME = tt-rss
-SPK_VERS = 20160102
-SPK_REV = 10
+SPK_VERS = 20170422
```

Comment:
The remove/install is hardly an option because you will lose data unless you do backups first.